### DISCUSSION TITLE
Update of website and documentation

### DISCUSSION THREAD

**Discussion by fipro78** [eclipse-nattable/nattable#117] [Announcements]
We recently migrated the [NatTable Website](https://eclipse.dev/nattable/) from PHP to Hugo. In that process we moved the documentation from the website to the [NatTable GitHub Wiki](https://github.com/eclipse-nattable/nattable/wiki).

We hope that with this move it is easier to extend and update the documentation in the future.

**Comment by lppedd** [comment]
Moving NatTable to GitHub and its documentation to the Wiki seems the most logical choice! Thank you!

I think you could get rid of the `NN_` prefix on each Wiki chapter.

**Reply by fipro78** [reply]
But then the chapters are sorted alphabetically on the right. At least if I understand it correctly. But there is of course room for improvement in that area.
